spec/lib/extensions/ar_region_spec.rb

```diff
@@ -125,4 +125,71 @@
       expect(ManageIQ::Providers::Vmware::InfraManager::Vm.new.split_id).to eq([ManageIQ::Providers::Vmware::InfraManager::Vm.my_region_number, nil])
     end
   end
+
+  describe ".my_region_number" do
+    before do
+      MiqDatabase.delete_all
+    end
```

Comment:
This should not be necessary...if you don't have an empty database at this point, something else is wrong.